.github/workflows/build.yml

```diff
@@ -0,0 +1,325 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  schedule:
+    - cron: "0 7 * * *"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  azCliVersion: 2.6.0
+  ref_javaee: 6addd99d8bc3f472e040f11c053a37e1ac370229
+  offerName: azure.websphere-traditional.image
+  userEmail: ${{ secrets.USER_EMAIL }}
+  userName: ${{ secrets.USER_NAME }}
+  unEntitledIbmUserId: ${{ secrets.UNENTITLED_IBM_USER_ID }}
+  unEntitledIbmPassword: ${{ secrets.UNENTITLED_IBM_USER_PWD }}
+  entitledIbmUserId: ${{ secrets.ENTITLED_IBM_USER_ID }}
+  entitledIbmPassword: ${{ secrets.ENTITLED_IBM_USER_PWD }}
+  vmName: was${{ github.run_id }}${{ github.run_number }}
+  vmAdminId: websphere
```

Comment:
Do we also need params for `wasadmin`?

Reply:
w@git4rk No, because the portion of the installation that requires `wasadmin` is not covered by this workflow.  That happens at VM provisioning time. This workflow is for stuff that happens in the process of creating the VM image. However, looking at

```
  vmAdminId: websphere
  vmAdminPassword: Secret123456
```

I believe these should be secrets.